frontend/packages/metal3-plugin/src/components/baremetal-hosts/dashboard/EventsCard.tsx

```diff
@@ -71,18 +86,42 @@ const EventsCard: React.FC<EventsCardProps> = ({
       }
     };
   }, [watchK8sResource, stopWatchK8sResource, machineName, namespace]);
+
+  const filter = getHostEventsFilter(obj, _.get(resources.machine, 'data') as MachineKind);
+
+  const inProgress = isHostInProgressState(obj);
+  const hostStatus = getBareMetalHostStatus(obj);
+
   return (
     <DashboardCard>
       <DashboardCardHeader>
-        <DashboardCardTitle>Events</DashboardCardTitle>
+        <DashboardCardTitle>Activity</DashboardCardTitle>
       </DashboardCardHeader>
       <DashboardCardBody>
-        <EventsBody
-          events={resources.events as FirehoseResult<EventKind[]>}
-          filter={(event) =>
-            hostEventsFilter(obj, _.get(resources.machine, 'data') as MachineKind, event)
-          }
-        />
+        <ActivityBody>
+          <div className="co-activity-card__ongoing-title">Ongoing</div>
+          <div className="co-activity-card__ongoing-body">
+            {inProgress ? (
+              <Activity timestamp={obj.status.lastUpdated}>
```

Comment:
the timestamp is supposed to show when the activity started, but `lastUpdated` can mean anything (any field got updated). I dont think you can detect when the node actually started doing something